mathics/core/expression.py

```diff
@@ -2137,7 +2137,7 @@ def __neg__(self) -> 'Rational':
 
     @property
     def is_zero(self) -> bool:
-        return self.numerator().is_zero and not self.denominator().is_zero()
+        return self.numerator().is_zero and not(self.denominator().is_zero)
 
```

Comment:
Should this be in a separate PR. It fixes a current bug and is not specific to asymptote, right?

Reply:
I put this because I got CI errors due to PossibleZeroQ. Please look at #1164.